.gitignore

```diff
@@ -56,6 +56,7 @@ typings/
 
 # dotenv environment variables file
 .env
+!frontend/.env*
```

Comment:
could you add a comment here to make it clear why we're excluding the frontend/.env? Something like "frontend env files shouldn't contain secrets" should be good!